bin/usgscsm_cam_test.cc

```diff
@@ -0,0 +1,64 @@
+// A tool to perform some basic tests and operations on a CSM camera model.
+// 
+// Functionality:
+//
+// - Load a CSM model in ISD format.
+//
+// Future functionality:
+// 
+// - Test projecting rays from the camera to ground and vice-versa.
+// - Load a CSM model state (stored in a .json file, just like
+//   an ISD model).
+// - Ability to export a CSM model in ISD format to a CSM model state file. 
+
+#include <usgscsm/UsgsAstroPlugin.h>
+#include <csm/RasterGM.h>
+
+#include <iostream>
+int main(int argc, char **argv) {
+
+  if (argc != 2) {
+    std::cerr << "Usage: " << argv[0] << " <model file>" << std::endl;
+    return 1;
+  }
+  
+  std::string model_file = argv[1];
+  csm::Isd isd(model_file);
+
+  std::vector<std::string> model_types = {"USGS_ASTRO_FRAME_SENSOR_MODEL",
```

Comment:
This is accessible from the plugin. See Plugin::getNumModels and Plugin::getModelName